<!doctype html><html lang=en-us><head><meta charset=utf-8><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><!--[if IEMobile]><meta http-equiv=cleartype content="on"><![endif]--><title>Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater - ACL Anthology</title><meta name=generator content="Hugo 0.68.3"><link href=/website/aclicon.ico rel="shortcut icon" type=image/x-icon><link rel=stylesheet href=/website/css/main.min.8976777c0832d068a49d330764e507857027f1efa3b8501cf349b0e2db7410fc.css media=screen><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.7.2/css/all.css integrity=sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr crossorigin=anonymous><link rel=stylesheet href=/website/css/academicons.min.css><meta content="Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater" name=citation_title><meta content="M. Fuzzen" name=citation_author><meta content="N. Harper" name=citation_author><meta content="H. Dhiyebi" name=citation_author><meta content="N. Srikanthan" name=citation_author><meta content="S. Hayat" name=citation_author><meta content="S. Peterson" name=citation_author><meta content="I. Yang" name=citation_author><meta content="J. X. Sun" name=citation_author><meta content="E. Edwards" name=citation_author><meta content="J. Giesy" name=citation_author><meta content="C. Mangat" name=citation_author><meta content="T. Graber" name=citation_author><meta content="R. E. Delatolla" name=citation_author><meta content="M. Servos" name=citation_author><meta content="Global Water Futures 2022" name=citation_conference_title><meta content="2022" name=citation_publication_date><meta content="https://mobehbooei.github.io/website/2022.GWF-1.18.pdf" name=citation_pdf_url><meta content="10.1101/2022.04.12.22273761" name=citation_doi><meta property="og:title" content="Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater"><meta property="og:image" content="https://mobehbooei.github.io/website/thumb/G22-1018.jpg"><meta property="og:image:alt" content="First page of paper PDF."><meta property="og:type" content="article"><meta property="og:site_name" content="ACL Anthology"><meta property="og:url" content="https://aclanthology.org/2022.GWF-1.18"><meta property="og:description" content="M. Fuzzen, N. Harper, H. Dhiyebi, N. Srikanthan, S. Hayat, S. Peterson, I. Yang, J. X. Sun, E. Edwards, J. Giesy, C. Mangat, T. Graber, R. E. Delatolla, M. Servos. Global Water Futures 2022. 2022."><link rel=canonical href=https://aclanthology.org/2022.GWF-1.18></head><body><nav class="navbar navbar-expand-sm navbar-light bg-light bg-gradient-light shadow-sm py-0 mb-3 mb-md-4 mb-xl-5"><div id=navbar-container class=container><a class=navbar-brand href=/website/><img src=/website/images/acl-logo.svg width=56 alt="ACL Logo">
<span class="d-none d-md-inline pl-md-2">ACL Anthology</span></a>
<button class=navbar-toggler type=button data-toggle=collapse data-target=#navbarSupportedContent aria-controls=navbarSupportedContent aria-expanded=false aria-label="Toggle navigation">
<span class=navbar-toggler-icon></span></button><div class="collapse navbar-collapse" id=navbarSupportedContent><ul class="navbar-nav mr-auto"><li class=nav-item><a class=nav-link href=/website/faq/>FAQ<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/website/info/corrections/>Corrections<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/website/info/contrib/>Submissions<span class=sr-only>(current)</span></a></li></ul><form class="form-inline my-2 my-lg-0" action=/website/search/? method=get><input id=acl-search-box class="form-control mr-sm-2" name=q type=search placeholder=Search... aria-label=Search>
<button class="btn btn-outline-primary" type=submit><i class="fas fa-search"></i></button></form></div></div></nav><div id=main-container class=container><section id=main><div><h2 id=title><a href=https://mobehbooei.github.io/website/2022.GWF-1.18.pdf>Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater</a></h2><p class=lead><a href=/website/people/M/M-Fuzzen/>M. Fuzzen</a>,
<a href=/website/people/N/N-Harper/>N. Harper</a>,
<a href=/website/people/H/H-Dhiyebi/>H. Dhiyebi</a>,
<a href=/website/people/N/N-Srikanthan/>N. Srikanthan</a>,
<a href=/website/people/S/S-Hayat/>S. Hayat</a>,
<a href=/website/people/S/S-Peterson/>S. Peterson</a>,
<a href=/website/people/I/I-Yang/>I. Yang</a>,
<a href=/website/people/J/J-X-Sun/>J. X. Sun</a>,
<a href=/website/people/E/E-Edwards/>E. Edwards</a>,
<a href=/website/people/J/J-Giesy/>J. Giesy</a>,
<a href=/website/people/C/C-Mangat/>C. Mangat</a>,
<a href=/website/people/T/T-Graber/>T. Graber</a>,
<a href=/website/people/R/R-E-Delatolla/>R. E. Delatolla</a>,
<a href=/website/people/M/M-Servos/>M. Servos</a></p></div><hr><div class="row acl-paper-details"><div class="col col-lg-10 order-2"><div class="card bg-light mb-2 mb-lg-3"><div class="card-body acl-abstract"><h5 class=card-title>Abstract</h5><span>Wastewater-based surveillance (WBS) has become an effective tool around the globe for indirect monitoring of COVID-19 in communities. Quantities of viral fragments of SARS-CoV-2 in wastewater are related to numbers of clinical cases of COVID-19 reported within the corresponding sewershed. Variants of Concern (VOCs) have been detected in wastewater by use of reverse transcription quantitative polymerase chain reaction (RT-qPCR) or sequencing. A multiplex RT-qPCR assay to detect and estimate the prevalence of multiple VOCs, including Omicron/Alpha, Beta, Gamma, and Delta, in wastewater RNA extracts was developed and validated. The probe-based multiplex assay, named N200, focuses on amino acids 199-202, a region of the N gene that contains several mutations that are associated with variants of SARS-CoV-2 within a single amplicon. Each of the probes in the N200 assay are specific to the targeted mutations and worked equally well in single- and multi-plex modes. To estimate prevalence of each VOC, the abundance of the targeted mutation was compared with a non-mutated region within the same amplified region. The N200 assay was applied to monitor frequencies of VOCs in wastewater extracts from six sewersheds in Ontario, Canada collected between December 1, 2021, and January 4, 2022. Using the N200 assay, the replacement of the Delta variant along with the introduction and rapid dominance of the Omicron variant were monitored in near real-time, as they occurred nearly simultaneously at all six locations. The N200 assay is robust and efficient for wastewater surveillance can be adopted into VOC monitoring programs or replace more laborious assays currently being used to monitor SARS-CoV-2 and its VOCs.</span></div></div><dl><dt>Anthology ID:</dt><dd>G22-1018</dd><dt>Volume:</dt><dd><a href=/website/volumes/G22-1/>Global Water Futures 2022</a></dd><dt>Month:</dt><dd></dd><dt>Year:</dt><dd>2022</dd><dt>Address:</dt><dd>Online</dd><dt>Venue:</dt><dd><a href=/website/venues/gwf/>GWF</a></dd><dt>SIG:</dt><dd></dd><dt>Publisher:</dt><dd>University of Waterloo</dd><dt>Note:</dt><dd></dd><dt>Pages:</dt><dd></dd><dt>Language:</dt><dd></dd><dt>URL:</dt><dd><a href=https://aclanthology.org/2022.GWF-1.18>https://aclanthology.org/2022.GWF-1.18</a></dd><dt>DOI:</dt><dd><a href=http://dx.doi.org/10.1101/2022.04.12.22273761 title="To the current version of the paper by DOI">10.1101/2022.04.12.22273761</a></dd><dt class=acl-button-row>Bibkey:</dt><dd class=acl-button-row><button type=button class="btn btn-clipboard-outside btn-secondary btn-sm d-none" data-clipboard-target=#citePaperBibkey><i class="far fa-clipboard"></i><span id=citePaperBibkey class="pl-2 text-monospace">Fuzzen-etal-2022-Multiplex</span></button></dd><dt>Cite (ACL):</dt><dd><span id=citeACL>M. Fuzzen, N. Harper, H. Dhiyebi, N. Srikanthan, S. Hayat, S. Peterson, I. Yang, J. X. Sun, E. Edwards, J. Giesy, C. Mangat, T. Graber, R. E. Delatolla, and M. Servos. 2022. <a href=https://aclanthology.org/2022.GWF-1.18>Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater</a>. In <i>Global Water Futures 2022</i>, Online. University of Waterloo.</span><button type=button class="btn btn-clipboard btn-secondary btn-sm d-none ml-2" data-clipboard-target=#citeACL><i class="far fa-clipboard"></i></button></dd><dt>Cite (Informal):</dt><dd><span id=citeRichText><a href=https://aclanthology.org/2022.GWF-1.18>Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater</a> (Fuzzen et al., GWF 2022)</span><button type=button class="btn btn-clipboard btn-secondary btn-sm d-none ml-2" data-clipboard-target=#citeRichText><i class="far fa-clipboard"></i></button></dd><dt class=acl-button-row>Copy Citation:</dt><dd class=acl-button-row><button type=button class="btn btn-clipboard-outside btn-secondary btn-sm d-none" data-clipboard-target=#citeBibtexContent><i class="far fa-clipboard pr-2"></i>BibTeX</button>
<button type=button class="btn btn-clipboard-outside btn-secondary btn-sm d-none" data-clipboard-target=#citeMarkdownContent><i class="far fa-clipboard pr-2"></i>Markdown</button>
<button type=button class="btn btn-clipboard-outside btn-secondary btn-sm d-none" data-clipboard-target=#citeModsContent><i class="far fa-clipboard pr-2"></i>MODS XML</button>
<button type=button class="btn btn-clipboard-outside btn-secondary btn-sm d-none" data-clipboard-target=#citeEndnoteContent><i class="far fa-clipboard pr-2"></i>Endnote</button>
<button type=button class="btn btn-secondary btn-sm" data-toggle=modal data-target=#citeModal>More optionsâ€¦</button></dd><dt>PDF:</dt><dd><a href=https://mobehbooei.github.io/website/2022.GWF-1.18.pdf>https://mobehbooei.github.io/website/2022.GWF-1.18.pdf</a></dd></dl></div><div class=acl-paper-link-block><a class="btn btn-primary" href=https://mobehbooei.github.io/website/2022.GWF-1.18.pdf title="Open PDF of 'Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater'"><i class="far fa-file-pdf"></i><span class=pl-2>PDF</span></a>
<a class="btn btn-secondary" title="Open dialog for exporting citations" data-toggle=modal data-target=#citeModal href=#><i class="fas fa-quote-left"></i><span class=pl-2>Cite</span></a>
<a class="btn btn-secondary" href="https://www.semanticscholar.org/search?q=Multiplex+RT-qPCR+assay+%28N200%29+to+detect+and+estimate+prevalence+of+multiple+SARS-CoV-2+Variants+of+Concern+in+wastewater" title="Search for 'Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater' on Semantic Scholar"><i class="ai ai-semantic-scholar"></i><span class="pl-sm-2 d-none d-sm-inline">Search</span></a></div></div><hr><div class="modal fade" id=citeModal tabindex=-1 role=dialog aria-labelledby=citeModalLabel aria-hidden=true><div class="modal-dialog modal-lg" role=document><div class=modal-content><div class=modal-header><h5 class=modal-title id=citeModalLabel>Export citation</h5><button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><ul class="nav nav-tabs mb-2" id=citeFormats role=tablist><li class=nav-item><a class="nav-link active" data-toggle=list href=#citeBibtex role=tab aria-controls=citeBibtex aria-selected=true>BibTeX</a></li><li class=nav-item><a class=nav-link data-toggle=list href=#citeMods role=tab aria-controls=citeMods aria-selected=false>MODS XML</a></li><li class=nav-item><a class=nav-link data-toggle=list href=#citeEndnote role=tab aria-controls=citeEndnote aria-selected=false>Endnote</a></li><li class=nav-item><a class=nav-link data-toggle=list href=#citeMarkdown role=tab aria-controls=citeMarkdown aria-selected=false>Preformatted</a></li></ul><div class=tab-content id=citeFormatsContent><div class="tab-pane active" id=citeBibtex role=tabpanel><pre id=citeBibtexContent class="bg-light border p-2" style=max-height:50vh>@inproceedings{Fuzzen-etal-2022-Multiplex,
    title = &#34;Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater&#34;,
    author = &#34;Fuzzen, M.  and
      Harper, N.  and
      Dhiyebi, H.  and
      Srikanthan, N.  and
      Hayat, S.  and
      Peterson, S.  and
      Yang, I.  and
      X. Sun, J.  and
      Edwards, E.  and
      Giesy, J.  and
      Mangat, C.  and
      Graber, T.  and
      E. Delatolla, R.  and
      Servos, M.&#34;,
    booktitle = &#34;Global Water Futures 2022&#34;,
    year = &#34;2022&#34;,
    address = &#34;Online&#34;,
    publisher = &#34;University of Waterloo&#34;,
    url = &#34;https://aclanthology.org/2022.GWF-1.18&#34;,
    doi = &#34;10.1101/2022.04.12.22273761&#34;,
    abstract = &#34;Wastewater-based surveillance (WBS) has become an effective tool around the globe for indirect monitoring of COVID-19 in communities. Quantities of viral fragments of SARS-CoV-2 in wastewater are related to numbers of clinical cases of COVID-19 reported within the corresponding sewershed. Variants of Concern (VOCs) have been detected in wastewater by use of reverse transcription quantitative polymerase chain reaction (RT-qPCR) or sequencing. A multiplex RT-qPCR assay to detect and estimate the prevalence of multiple VOCs, including Omicron/Alpha, Beta, Gamma, and Delta, in wastewater RNA extracts was developed and validated. The probe-based multiplex assay, named N200, focuses on amino acids 199-202, a region of the N gene that contains several mutations that are associated with variants of SARS-CoV-2 within a single amplicon. Each of the probes in the N200 assay are specific to the targeted mutations and worked equally well in single- and multi-plex modes. To estimate prevalence of each VOC, the abundance of the targeted mutation was compared with a non-mutated region within the same amplified region. The N200 assay was applied to monitor frequencies of VOCs in wastewater extracts from six sewersheds in Ontario, Canada collected between December 1, 2021, and January 4, 2022. Using the N200 assay, the replacement of the Delta variant along with the introduction and rapid dominance of the Omicron variant were monitored in near real-time, as they occurred nearly simultaneously at all six locations. The N200 assay is robust and efficient for wastewater surveillance can be adopted into VOC monitoring programs or replace more laborious assays currently being used to monitor SARS-CoV-2 and its VOCs.&#34;,
}
</pre><div class="modal-footer pb-1"><a class="btn btn-secondary" href=/website/G22-1018.bib><i class="fas fa-download pr-2"></i>Download as File</a>
<button class="btn btn-clipboard btn-primary d-none" data-clipboard-target=#citeBibtexContent><i class="far fa-clipboard pr-2"></i>Copy to Clipboard</button></div></div><div class=tab-pane id=citeMods role=tabpanel><pre id=citeModsContent class="bg-light border p-2" style=max-height:50vh>ï»¿&lt;?xml version=&#34;1.0&#34; encoding=&#34;UTF-8&#34;?&gt;
&lt;modsCollection xmlns=&#34;http://www.loc.gov/mods/v3&#34;&gt;
&lt;mods ID=&#34;Fuzzen-etal-2022-Multiplex&#34;&gt;
    &lt;titleInfo&gt;
        &lt;title&gt;Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater&lt;/title&gt;
    &lt;/titleInfo&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;M&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Fuzzen&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;N&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Harper&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;H&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Dhiyebi&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;N&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Srikanthan&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;S&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Hayat&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;S&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Peterson&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;I&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Yang&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;J&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;X. Sun&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;E&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Edwards&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;J&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Giesy&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;C&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Mangat&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;T&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Graber&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;R&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;E. Delatolla&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;name type=&#34;personal&#34;&gt;
        &lt;namePart type=&#34;given&#34;&gt;M&lt;/namePart&gt;
        &lt;namePart type=&#34;family&#34;&gt;Servos&lt;/namePart&gt;
        &lt;role&gt;
            &lt;roleTerm authority=&#34;marcrelator&#34; type=&#34;text&#34;&gt;author&lt;/roleTerm&gt;
        &lt;/role&gt;
    &lt;/name&gt;
    &lt;originInfo&gt;
        &lt;dateIssued&gt;2022&lt;/dateIssued&gt;
    &lt;/originInfo&gt;
    &lt;typeOfResource&gt;text&lt;/typeOfResource&gt;
    &lt;relatedItem type=&#34;host&#34;&gt;
        &lt;titleInfo&gt;
            &lt;title&gt;Global Water Futures 2022&lt;/title&gt;
        &lt;/titleInfo&gt;
        &lt;originInfo&gt;
            &lt;publisher&gt;University of Waterloo&lt;/publisher&gt;
            &lt;place&gt;
                &lt;placeTerm type=&#34;text&#34;&gt;Online&lt;/placeTerm&gt;
            &lt;/place&gt;
        &lt;/originInfo&gt;
        &lt;genre authority=&#34;marcgt&#34;&gt;conference publication&lt;/genre&gt;
    &lt;/relatedItem&gt;
    &lt;abstract&gt;Wastewater-based surveillance (WBS) has become an effective tool around the globe for indirect monitoring of COVID-19 in communities. Quantities of viral fragments of SARS-CoV-2 in wastewater are related to numbers of clinical cases of COVID-19 reported within the corresponding sewershed. Variants of Concern (VOCs) have been detected in wastewater by use of reverse transcription quantitative polymerase chain reaction (RT-qPCR) or sequencing. A multiplex RT-qPCR assay to detect and estimate the prevalence of multiple VOCs, including Omicron/Alpha, Beta, Gamma, and Delta, in wastewater RNA extracts was developed and validated. The probe-based multiplex assay, named N200, focuses on amino acids 199-202, a region of the N gene that contains several mutations that are associated with variants of SARS-CoV-2 within a single amplicon. Each of the probes in the N200 assay are specific to the targeted mutations and worked equally well in single- and multi-plex modes. To estimate prevalence of each VOC, the abundance of the targeted mutation was compared with a non-mutated region within the same amplified region. The N200 assay was applied to monitor frequencies of VOCs in wastewater extracts from six sewersheds in Ontario, Canada collected between December 1, 2021, and January 4, 2022. Using the N200 assay, the replacement of the Delta variant along with the introduction and rapid dominance of the Omicron variant were monitored in near real-time, as they occurred nearly simultaneously at all six locations. The N200 assay is robust and efficient for wastewater surveillance can be adopted into VOC monitoring programs or replace more laborious assays currently being used to monitor SARS-CoV-2 and its VOCs.&lt;/abstract&gt;
    &lt;identifier type=&#34;citekey&#34;&gt;Fuzzen-etal-2022-Multiplex&lt;/identifier&gt;
    &lt;identifier type=&#34;doi&#34;&gt;10.1101/2022.04.12.22273761&lt;/identifier&gt;
    &lt;location&gt;
        &lt;url&gt;https://aclanthology.org/2022.GWF-1.18&lt;/url&gt;
    &lt;/location&gt;
    &lt;part&gt;
        &lt;date&gt;2022&lt;/date&gt;
    &lt;/part&gt;
&lt;/mods&gt;
&lt;/modsCollection&gt;
</pre><div class="modal-footer pb-1"><a class="btn btn-secondary" href=/website/G22-1018.xml><i class="fas fa-download pr-2"></i>Download as File</a>
<button class="btn btn-clipboard btn-primary d-none" data-clipboard-target=#citeModsContent><i class="far fa-clipboard pr-2"></i>Copy to Clipboard</button></div></div><div class=tab-pane id=citeEndnote role=tabpanel><pre id=citeEndnoteContent class="bg-light border p-2" style=max-height:50vh>ï»¿%0 Conference Proceedings
%T Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater
%A Fuzzen, M.
%A Harper, N.
%A Dhiyebi, H.
%A Srikanthan, N.
%A Hayat, S.
%A Peterson, S.
%A Yang, I.
%A X. Sun, J.
%A Edwards, E.
%A Giesy, J.
%A Mangat, C.
%A Graber, T.
%A E. Delatolla, R.
%A Servos, M.
%S Global Water Futures 2022
%D 2022
%I University of Waterloo
%C Online
%F Fuzzen-etal-2022-Multiplex
%X Wastewater-based surveillance (WBS) has become an effective tool around the globe for indirect monitoring of COVID-19 in communities. Quantities of viral fragments of SARS-CoV-2 in wastewater are related to numbers of clinical cases of COVID-19 reported within the corresponding sewershed. Variants of Concern (VOCs) have been detected in wastewater by use of reverse transcription quantitative polymerase chain reaction (RT-qPCR) or sequencing. A multiplex RT-qPCR assay to detect and estimate the prevalence of multiple VOCs, including Omicron/Alpha, Beta, Gamma, and Delta, in wastewater RNA extracts was developed and validated. The probe-based multiplex assay, named N200, focuses on amino acids 199-202, a region of the N gene that contains several mutations that are associated with variants of SARS-CoV-2 within a single amplicon. Each of the probes in the N200 assay are specific to the targeted mutations and worked equally well in single- and multi-plex modes. To estimate prevalence of each VOC, the abundance of the targeted mutation was compared with a non-mutated region within the same amplified region. The N200 assay was applied to monitor frequencies of VOCs in wastewater extracts from six sewersheds in Ontario, Canada collected between December 1, 2021, and January 4, 2022. Using the N200 assay, the replacement of the Delta variant along with the introduction and rapid dominance of the Omicron variant were monitored in near real-time, as they occurred nearly simultaneously at all six locations. The N200 assay is robust and efficient for wastewater surveillance can be adopted into VOC monitoring programs or replace more laborious assays currently being used to monitor SARS-CoV-2 and its VOCs.
%R 10.1101/2022.04.12.22273761
%U https://aclanthology.org/2022.GWF-1.18
%U https://doi.org/10.1101/2022.04.12.22273761

</pre><div class="modal-footer pb-1"><a class="btn btn-secondary" href=/website/G22-1018.endf><i class="fas fa-download pr-2"></i>Download as File</a>
<button class="btn btn-clipboard btn-primary d-none" data-clipboard-target=#citeEndnoteContent><i class="far fa-clipboard pr-2"></i>Copy to Clipboard</button></div></div><div class=tab-pane id=citeMarkdown role=tabpanel><h5>Markdown (Informal)</h5><p id=citeMarkdownContent class="text-monospace small bg-light border p-2">[Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater](https://aclanthology.org/2022.GWF-1.18) (Fuzzen et al., GWF 2022)</p><ul class=mt-2><li><a href=https://aclanthology.org/2022.GWF-1.18>Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater</a> (Fuzzen et al., GWF 2022)</li></ul><h5>ACL</h5><ul class=mt-2><li id=citeACLstyleContent>M. Fuzzen, N. Harper, H. Dhiyebi, N. Srikanthan, S. Hayat, S. Peterson, I. Yang, J. X. Sun, E. Edwards, J. Giesy, C. Mangat, T. Graber, R. E. Delatolla, and M. Servos. 2022. <a href=https://aclanthology.org/2022.GWF-1.18>Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater</a>. In <i>Global Water Futures 2022</i>, Online. University of Waterloo.</li></ul><div class="modal-footer pb-1"><button type=button class="btn btn-clipboard btn-primary d-none" data-clipboard-target=#citeMarkdownContent><i class="far fa-clipboard pr-2"></i>Copy Markdown to Clipboard</button>
<button type=button class="btn btn-clipboard btn-primary d-none" data-clipboard-target=#citeACLstyleContent><i class="far fa-clipboard pr-2"></i>Copy ACL to Clipboard</button></div></div></div></div></div></div></div></section></div><footer class="bg-gradient-light py-2 py-xl-3 mt-3 mt-md-4 mt-xl-5"><div class=container><p class="text-muted small px-1"><span class="float-right mt-2 ml-2"><a rel=license href=http://creativecommons.org/licenses/by/4.0/><img alt="Creative Commons License" style=border-width:0 src=https://i.creativecommons.org/l/by/4.0/88x31.png></a></span>
ACL materials are Copyright Â©&nbsp;1963&ndash;2022 ACL; other materials are copyrighted by their respective copyright holders. Materials prior to 2016 here are licensed under the <a href=https://creativecommons.org/licenses/by-nc-sa/3.0/>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 International License</a>. Permission is granted to make copies for the purposes of teaching and research. Materials published in or after 2016 are licensed on a <a href=https://creativecommons.org/licenses/by/4.0/>Creative Commons Attribution 4.0 International License</a>.</p><p class="text-muted small px-1">The ACL Anthology is managed and built by the <a href=/website/info/credits/>ACL Anthology team</a> of volunteers.</p><p class="text-muted small px-1"><i>Site last built on 17 November 2022 at 01:58 UTC with <a href=https://github.com/acl-org/acl-anthology/tree/51de55f083dd8e08971d5f9d4a69e24f90b89cf2>commit 51de55f0</a>.</i></p></div></footer><script src=https://code.jquery.com/jquery-3.3.1.slim.min.js integrity=sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js integrity=sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut crossorigin=anonymous></script><script src=https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js integrity=sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k crossorigin=anonymous></script><script>$(function(){$('[data-toggle="tooltip"]').tooltip();if($("#toggle-all-abstracts")){$("#toggle-all-abstracts").click(function(){var target=$("#toggle-all-abstracts");target.attr("disabled",true);if(target.attr("data-toggle-state")=="hide"){$(".abstract-collapse").collapse('show');target.attr("data-toggle-state","show");}else{$(".abstract-collapse").collapse('hide');target.attr("data-toggle-state","hide");}
target.attr("disabled",false);});$("#toggle-all-abstracts").attr("disabled",false);}})</script><script src=/website/js/clipboard.min.js></script><script>$(document).ready(function(){if(ClipboardJS.isSupported()){success_fn=function(e){var src=$(e.trigger);src.toggleClass("btn-success");src.children("i").toggleClass("far fa-clipboard fas fa-clipboard-check");e.clearSelection();setTimeout(function(){src.toggleClass("btn-success");src.children("i").toggleClass("far fa-clipboard fas fa-clipboard-check");},2000);};var clipboard=new ClipboardJS(".btn-clipboard");clipboard.on('success',success_fn);$(".btn-clipboard").removeClass("d-none");var clipboard_outside=new ClipboardJS(".btn-clipboard-outside",{text:function(trigger){var target=trigger.getAttribute("data-clipboard-target");return $(target).text();}});clipboard_outside.on('success',success_fn);$(".btn-clipboard-outside").removeClass("d-none");}});</script></body></html>